packages/web/src/components/keyboard-shortcuts-dialog.tsx

```diff
@@ -45,6 +45,7 @@ export default function KeyboardShortcutsDialog({
               <ShortcutRow keys={['alt', '↑']} description="move lines up" />
               <ShortcutRow keys={['alt', '↓']} description="move lines down" />
               <ShortcutRow keys={['shift', 'alt', 'f']} description="format code using Prettier" />
+              <ShortcutRow keys={['alt', 'click']} description="Go to definition" />
```

Comment:
nano nit but the other descriptions all use lower case. Don't care either way but would prefer consistency

Reply:
Oops, fixed